CodesmithProjects/TrailBlazers
# /api/trails path and /api/moreInfo path added to the server.

## Overview

Issue Type


Description
Added backend server paths to respond to aforementioned URL's.

Ticket Item

FEATURE #1: SVC
FEATURE #2

Steps to Validate Feature

1. Go to 'https://www.postman.com/'
2. Click on 'Postman API'
3. Click on '+' tab to create a new request
4. Enter 'http://localhost:4000/api/trails/ followed by desired zip code
5. Click the "Send" button
6. Verify response object at Response section at the bottom of the page.

Previous behavior

A 404 error would appear for requests to the aforementioned paths.

Expected behavior

Expected to get response objects with information from trail APIs

Screenshots & Videos

Example screenshot
![bRSRG1M](https://user-images.githubusercontent.com/118780080/230735749-a9f56adf-eaf1-4913-9a67-eb26a84a4fcd.png)

Additional context

Added similar functionality to the /api/moreInfo/ path. A valid park ID is needed after the last forward slash.
